src/main/java/com/systemmeltdown/robotlib/subsystems/drive/SingleSpeedTalonDriveSubsystem.java

```diff
@@ -0,0 +1,73 @@
+package com.systemmeltdown.robotlib.subsystems.drive;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+public class SingleSpeedTalonDriveSubsystem extends SkidSteerDriveSubsystem {
+    private WPI_TalonSRX m_rightMaster;
+    private WPI_TalonSRX m_leftMaster;
+    private WPI_TalonSRX[] m_rightSlaves;
+    private WPI_TalonSRX[] m_leftSlaves;
+
+    public SingleSpeedTalonDriveSubsystem(
+        TalonIDGroup rightTalonIDs,
+        TalonIDGroup leftTalonIDs) {
```

Comment:
I'm not sure having a class solely for the commuting of ids is better than just giving arrays of ids here.

I think if we have a small class to group talons, it should be more functional and manage the WPI_TalonSRX objects and their configs.

Also, I'd rather not have configuration options like inversion in the constructor. I'd rather pass those through the `configure()` method which is designed for this.

Reply:
When you mention the `configure()` method, I assume you're talking about the same one you mentioned overriding earlier. What object is the method a member of? For now, I've added a `configure(boolean isInverted)` method to the newly made TalonGroup.

Reply:
This configure in SkidSteerDriveSubsystem:
https://github.com/frc2357/robotlib2357/blob/5ebd2ca62e1007a9337ccca889c899ae6c8a553a/src/main/java/com/systemmeltdown/robotlib/subsystems/drive/SkidSteerDriveSubsystem.java#L43

It's designed to be overridden, pluck out the settings from the map you care about, then pass on the map to the superclass.

Reply:
I would suggest adding configuration values to the map like `"invertLeft"` = `true` and `"invertRight"` = `true`

The real advantage of this is you can keep all these configuration options together in one place and in one function call.

In the future, we'll add more configuration options to this map, like specific ones for the TalonSRX for ramping, PIDF values, etc.